docs/index.md

```diff
@@ -94,6 +94,7 @@ Parsers for these languages are in development:
 * [Sourcepawn](https://github.com/nilshelmig/tree-sitter-sourcepawn)
 * [Swift](https://github.com/tree-sitter/tree-sitter-swift)
 * [SQL](https://github.com/m-novikov/tree-sitter-sql)
+* [LALRPOP](https://github.com/traxys/tree-sitter-lalrpop)
```

Comment:
Thank you for adding support for this parser! One small nit: do you mind alphabetizing this list? Happy to re-review once you've got that pushed up. 😄

Reply:
It should now be in correct order!